models/fivetran_log__mar_table_history.sql

```diff
@@ -27,50 +27,37 @@ ordered_mar as (
         schema_name,
         table_name,
         destination_id,
-        measured_date,
         measured_month,
-        incremental_rows,
-        coalesce(case when lower(free_type) = 'paid'
+        max(measured_date) as last_measured_at,
+        sum(incremental_rows) as incremental_rows,
+        sum(coalesce(case when lower(free_type) = 'paid'
             then incremental_rows
-        end, 0) as paid_monthly_active_rows,
-        coalesce(case when lower(free_type) != 'paid'
+        end, 0)) as paid_monthly_active_rows,
+        sum(coalesce(case when lower(free_type) != 'paid'
             then incremental_rows
-        end, 0) as free_monthly_active_rows
+        end, 0)) as free_monthly_active_rows,
+        (free_monthly_active_rows + paid_monthly_active_rows) as total_monthly_active_rows
 
     from incremental_mar
-
-),
-
-latest_mar as (
-    select 
-        connector_name,
-        schema_name,
-        table_name,
-        destination_id,
-        measured_month,
-        date(measured_date) as last_measured_at,
-        free_monthly_active_rows,
-        paid_monthly_active_rows,
-        (free_monthly_active_rows + paid_monthly_active_rows) as total_monthly_active_rows
-    
-    from ordered_mar
```

Comment:
Unfortunately, we do need to keep this cte in the model. The recursive ability of Snowflake does not exist in other warehouses. Therefore, if we were to go forward with removing the cte, other warehouses would be prompted with a "free_monthly_active_rows field does not exist".

As such, I would recommend we bring this cte back and move the `total_monthly_active_rows` calculation to this cte.